### ISSUE TITLE
Channel version crashing while loading (splash screen)

### ISSUE THREAD

**Issue by carcaschoi** [V10lator/NUSspli#37]
App version: v1.31 -> v1.32
The music still playing and stuck on the splash screen after the in-app update

**Comment by carcaschoi** [comment]
![Uploading 16253196949277789284435519846730.jpg…]()

**Comment by carcaschoi** [comment]
Photo
https://lewd.pics/p/yRXR.jpg

**Comment by carcaschoi** [comment]
Overwrite installation doesnot work too. I need to delete it and install a newer version of NUSspli channel version

**Comment by V10lator** [comment]
Thanks for the report. One of my beta testers was able to confirm the issue but he's injured right now, so he can't provide the needed logs. We're working on it through.

**Comment by V10lator** [comment]
Another report of this:

> I was using the stable versions and it worked great but oddly I downloaded a few games and all is fine but now oddly NUSspli won't open at all no errors and can't pull logs

**Comment by V10lator** [comment]
@carcaschoi In case you want to help by providing logs: Use the DEBUG build of NUSspli and this: https://github.com/dimok789/loadiine_gx2/raw/master/udp_debug_reader/UdpDebugReader.exe

You just need to run the exe on your PC, then start NUSspli and it should show a log.

**Comment by carcaschoi** [comment]
I try online update from 1.32 to 1.33 and it solve the bug now

**Comment by V10lator** [comment]
Please keep this open none the less as others are still experiencing this.

**Comment by carcaschoi** [comment]
> 請繼續保持開放，因為其他人仍在經歷這種情況。

I find a new bug🤔. I will post it here tomorrow

**Comment by carcaschoi** [comment]
I try online update from 1.32 to 1.33, it stuck on the main screen.

Then I try to overwrite install debug channel version of 1.33 , and it stuck on the loading screen

**Comment by carcaschoi** [comment]
> @carcaschoi In case you want to help by providing logs: Use the DEBUG build of NUSspli and this: https://github.com/dimok789/loadiine_gx2/raw/master/udp_debug_reader/UdpDebugReader.exe
> 
> You just need to run the exe on your PC, then start NUSspli and it should show a log.

I dont know how to use

**Comment by V10lator** [comment]
> I try online update from 1.32 to 1.33, it stuck on the main screen.

Sounds like a different bug, so please open a new bug report for it.

> I dont know how to use

1. Download https://github.com/dimok789/loadiine_gx2/raw/master/udp_debug_reader/UdpDebugReader.exe to your PC.
2. Double click it - a window should open. Keep it open.
3. Download and install https://github.com/V10lator/NUSspli/releases/download/v1.33/NUSspli-1.33-Channel-DEBUG.zip to your Wii U.
4. Run the newly installed NUSspli version.
5. Have a look at the window on your PC.

**Comment by carcaschoi** [comment]
Not work for me because it cannot enter to this screen
https://lewd.pics/p/L9La.png

**Comment by V10lator** [comment]
Your picture shows the loading screen, not the splash.

**Comment by carcaschoi** [comment]
> Your picture shows the loading screen, not the splash.

Yes, I know. The app stuck on the splash screen

**Comment by V10lator** [comment]
So why are ou showing the loading screen? I'm horribly onfused now. Please keep each bug at its own report and don't mix.

**Comment by carcaschoi** [comment]
> So why are <ou showing the loading screen? I'm horribly confused now. Please keep each bug at its own report and don't mix.

I said I cannot enter to the screen that show on the picture😅. So I cannot dump the log of the app

**Comment by V10lator** [comment]
I see, sorry for me getting confused.

**Comment by V10lator** [comment]
Are you sure you can't grab a log? The log is one of the first things being initialized, long before the GPU. It's the 5. line of code executed:
```
	initStatus();
	initASAN();
	OSThread *mainThread = OSGetCurrentThread();
	OSSetThreadName(mainThread, "NUSspli");
	debugInit();
	debugPrintf("NUSspli " NUSSPLI_VERSION);
```

**Comment by carcaschoi** [comment]
> Are you sure you can't grab a log? The log is one of the first things being initialized, long before the GPU. It's the 5. line of code executed:
> 
> ```
> 	initStatus();
> 	initASAN();
> 	OSThread *mainThread = OSGetCurrentThread();
> 	OSSetThreadName(mainThread, "NUSspli");
> 	debugInit();
> 	debugPrintf("NUSspli " NUSSPLI_VERSION);
> ```

I am sure that I can't grap the log

**Comment by V10lator** [comment]
Just a shoot into the blue but please test https://public.nbg01.v10lator.de/NUSspli/NUSspli-1.34-ALPHA3-Channel-DEBUG.zip

**Comment by carcaschoi** [comment]
> Just a shoot into the blue but please test https://public.nbg01.v10lator.de/NUSspli/NUSspli-1.34-ALPHA3-Channel-DEBUG.zip

I try to overwrite with 1.34 alpha3 and the problem is the same
no log can be grap

**Comment by V10lator** [comment]
As told on Discord: Part of it will be fixed with 1.34, the other part is a CafeOS bug, so Nintendo would need to fix it. I can't do anything when it crashes before my code gets executed.
